docs/source/modules/damping_driver.rst

```diff
@@ -0,0 +1,139 @@
+damping_driver.f90
+======================
+
+Summary
+-------
+The ``damping driver`` module is called by the ``idealized_moist_phys`` module by setting ``do_damping`` to true. It controls the upper level momentum damping in Isca. It controls 4 functions:
```

Comment:
Maybe, remove the `` around damping driver, depending on our convention.
Suggested edit: "It controls the upper level momentum damping in Isca, and has 4 functions:"

Reply:
I've added a _ between the two words so I think that's right now? If we change convection I'll take them all away. I've kept the sentence about being called by IMP as i thought people might want to know where to find the switch. Does that make sense?